recipe/meta.yaml

```diff
@@ -39,16 +36,16 @@ test:
     - xz.exe --help    # [win]
     - unxz.exe --help  # [win]
     - test -f ${PREFIX}/include/lzma.h                          # [unix]
+    - test -f ${PREFIX}/lib/cmake/liblzma/liblzma-targets.cmake # [unix]
```

Comment:
I would recommend to test `liblzma-config.cmake` instead of `targets` since `targets` is kind of an "implementation" detail. Basically, `targets` gets loaded by `config`. What cmake really searches for is the `config` file, not the `targets`.